documentation/docs/11-ssr-and-javascript.md

```diff
@@ -0,0 +1,77 @@
+---
+title: SSR and JavaScript
+---
+
+By default, SvelteKit will server-render pages on-demand, then **hydrate** the rendered HTML in the client with an interactive Svelte app while initialising a **router** that takes over subsequent navigations.
```

Comment:
Similarly to how I had an opinion about 'on-the-fly' in the SvelteKit announcement blog post, I have less-strong opinions about 'on-demand' here. I think I'd personally say 'an on-demand rendering' but 'it does this rendering on demand'.

Reply:
thanks — am normally anal about grammar and punctuation but i seem to have a real blind spot about this. 'on demand' is correct